__tests__/app.test.js

```diff
@@ -277,5 +276,39 @@ describe('GET /api/articles/:article_id/comments', () => {
 			});
 	});
 });
-
-
+describe('POST /api/articles/:article_id/comments', () => {
+	test('201 - Created : Add a new comment', () => {
+		return request(app)
+			.post('/api/articles/2/comments')
+			.send({ username: 'icellusedkars', body: 'New comment' })
+			.expect(201)
+			.then((res) => {
+				expect(res.body.comment).toEqual({
+					article_id: 2,
+					author: 'icellusedkars',
+					body: 'New comment',
+					comment_id: 19,
+					created_at: expect.any(String),
+					votes: 0,
+				});
+			});
+	});
+	test('400 - Bad request : When invalid article_id is given', () => {
+		return request(app)
+			.post('/api/articles/invalid_id/comments')
+			.send({ username: 'icellusedkars', body: 'New comment' })
+			.expect(400)
+			.then((res) => {
+				expect(res.body.msg).toBe('Bad request');
+			});
+	});
+	test('400 - Bad request : Request with empty comment', () => {
+		return request(app)
+			.post('/api/articles/2/comments')
+			.send({ username: 'icellusedkars', body: '' })
+			.expect(400)
+			.then((res) => {
+				expect(res.body.msg).toBe('Bad request');
+			});
```

Comment:
Good tests! You're just missing one error handler for when there are missing keys in the POST body (ie. no username, or no body, or both).